homeassistant/components/schlage/select.py

```diff
@@ -0,0 +1,78 @@
+"""Platform for Schlage select integration."""
+
+from __future__ import annotations
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LockData, SchlageDataUpdateCoordinator
+from .entity import SchlageEntity
+
+_DESCRIPTIONS = (
+    SelectEntityDescription(
+        key="auto_lock_time",
+        translation_key="auto_lock_time",
+        entity_category=EntityCategory.CONFIG,
+        # valid values are from Schlage UI and also https://github.com/dknowles2/pyschlage/blob/72f84dc12f71355e89ad4c53352386f46d0b577d/pyschlage/lock.py#L387
```

Comment:
Please keep lines under 88 characters

Reply:
The link is over 80 characters. What do you recommend?

Reply:
You can always remove it, when people try to remove it and test it out they will find out it doesn't work I guess

Reply:
Sure. Updated the comment